roles/kafka_connect/tasks/zos_connect_plugins.yml

```diff
@@ -56,10 +56,8 @@
       - "confluentinc-kafka-connect-ibmmq*"
   register: kafka_connect_ibm_mq_plugins
 
-- name: Copy MQ libs under MQ plugin folders
-  shell: "cp {{ kafka_connect_ibm_mqm_lib_path }}/*.jar {{item}}/lib"
-  loop: "{{ kafka_connect_ibm_mq_plugins.files | map(attribute='path') | list }}"
-
+- name: Copy MQ libs under JDK ext
```

Comment:
You  can now delete the previous step as well, the one where we register `kafka_connect_ibm_mq_plugins` variable, since that variable was only used in this step